recipes/steem/meta.yaml

```diff
@@ -0,0 +1,100 @@
+{% set name = "steem" %}
+{% set version = "0.18.103" %}
+{% set sha256 = "215A3A0E87151428AB71AB13FBEAB353F1FFF3F1A3EB082C5A4D3B2C7581D5D5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/07/7e/142fb6db1e14db39e658832b9d1b7f493d2e856ecd9c17f08490f85bf71e/steem-{{ version }}.tar.gz
+  sha256: {{ sha256|lower }}
+
+build:
+  number: 0
+  skip: True  # [py<36]
+  script: python -m pip install --no-deps --ignore-installed .
+
```

Comment:
You need to declare entry points (cli stuff) in the build https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points

Reply:
Thanks for the tip.
I added them.